components/items/ItemsGrid/ItemsGridImage.vue

```diff
@@ -121,7 +121,9 @@ const onClickShoppingCart = () => {
 .in-cart-border {
   @include ktheme() {
     outline: 2px solid theme('k-blue') !important;
+    outline-offset: -1px;
     border-color: transparent !important;
+    background-color: theme('blue-light-cards');
```

Comment:
This doesn't belong under `in-cart-border` clasd

Reply:
what do you mean?

Reply:
![image](https://github.com/kodadot/nft-gallery/assets/22791238/325ff19b-085e-413b-910a-b28ab28ba66c)

class named `in-cart-border` should not affect `background` properties

so, change the name of the class or separate the background into a new selector